account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -712,6 +706,21 @@ def _find_accounts(self, mapping_accounts):
             acc_templ_criteria += [
                 '&', ('parent_id', 'child_of', [root_account_id]),
                 ('chart_template_id', '=', False)]
+        if chart_template.parent_id:
```

Comment:
This should be recursive: you can have more a grand-parent, grand-grand-parent, and so.